### PR TITLE
Search Results Page

### PR DESCRIPTION
**Pull request by sampark413**
## Description
A few sentences describing the overall goals of the pull request's commits.

Related issues: #<number>
Related PRs: #<number>

### CODE REVIEW

File: .idea/GroceryBuddy.iml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Pezhin) [inline .idea/GroceryBuddy.iml]: Probably want to set .gitignore to ignore all the files in .idea/

Comment (davidenwang) [reply .idea/GroceryBuddy.iml]: Good suggestion we'll add that in. Hopefully, this doesn't happen in the future as we are using vscode and not IDEA as the ide for this project.

File: frontend/components/ItemView.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  static navigationOptions = {
+    title: 'Item Details',
+  };
+
+  render() {
+    const { params } = this.props.navigation.state;
+    var item = params.item;
+    var name = item.name;
+    var price = '$' + item.price;
+    //var description = item.description;
```
Comment (davidenwang) [inline frontend/components/ItemView.js]: remove commented code and use 'let' 👍

File: frontend/pages/SearchResultsPage.js [hunk 3]
```diff
@@ -1,30 +1,124 @@
...
+      })
+      console.log(result);
+      this.props.navigation.navigate(
+       'Results', {searchResults: result.searchResults}
+      );
+    }
+
+    componentDidMount(){
+      const keyword = this.props.navigation.getParam("keyword", "");
+      //this.searchResults(keyword);
```
Comment (davidenwang) [inline frontend/pages/SearchResultsPage.js]: can ❌ comments

**Review by davidenwang — CHANGES_REQUESTED** [review]
good stuff so far just make some small changes

File: frontend/pages/SearchResultsPage.js [hunk 4]
```diff
@@ -1,30 +1,124 @@
...
+        console.log("Pressed row: "+index);
+    };
 
     render() {
+        const { params } = this.props.navigation.state;
+        console.log(this.state.searchResults);
         return (
-            <View style={styles.container}>
-                <Text>Search Results Page</Text>
+          <View style = {{flex:1}}>
```
Comment (davidenwang) [inline frontend/pages/SearchResultsPage.js]: oh small thing just extract styles into the stylesheet at the bottom

**Review by davidenwang — APPROVED** [review]
Good stuff!

**Pull request merged**